exercises/calculator.cpp

```diff
@@ -10,3 +10,44 @@
   Multiplication: 8
   Division: 2
 */
+
+#include <iostream>
+using namespace std;
+
+
+int Sum(int a,int b){
+return a+b;
+}
+
+int Difference(int a,int b){
+    return a-b;
+}
+
+int Multiplication(int a,int b){
+    return a*b;
+}
+
+int Division(int a,int b){
+    return a/b;
+}
+
+
+int main(){
+    cout<<"-------------Welcome to calculator---------------------"<<endl;
+    cout<<"Chose your operation \n Press 1 to Sum \n Press 2 to Difference \n Press 3 to 
+Multiplicate \n Press 4 to Division"<<endl;
```

Comment:
In questa riga hai inserito un 'a capo' e quindi il compilatore si lamenta perchè non trova il "fine istruzione". Se vuoi scrivere codice multiriga, quindi rimanendo nella stessa istruzione (in questo caso `cout` devi inserire un `\` a fine riga per indicare al compilatore "Il codice prosegue sotto, quindi non è un errore"

Esempio: (nota il `\` alla fine della riga)

```cpp
cout << "Chose your operation \n Press 1 to Sum \n Press 2 to Difference \n Press 3 to \
        Multiplicate \n Press 4 to Division "<<endl;
```